libc-test/Cargo.toml

```diff
@@ -108,18 +108,7 @@ harness = true
 
 [lints.rust]
 # FIXME(cleanup): make ident usage consistent in each file
+#   Switching this to "warn" causes issues with generated code
 unused_qualifications = "allow"
 
 [lints.clippy]
```

Comment:
I think this can be removed now

Reply:
i tried re-setting this to `warn`, but still same issue with the generated code. Perhaps will clean it up in some futture PR